apps/transformers/lib/transformations/arithmetic_add.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Transformers.ArithmeticAdd do
+  @behaviour Transformation
+
+  alias Transformers.Validations.NotBlank
+  alias Transformers.Validations.ValidationStatus
+
+  @addends "addends"
+  @target_field "targetField"
+
+  @impl Transformation
+  def transform(payload, parameters) do
+    with {:ok, [addends, target_field]} <- validate(parameters),
+         {:ok, sum} <- sumValues(addends, payload) do
+      {:ok, payload |> Map.put(target_field, sum)}
+    else
+      {:error, reason} -> {:error, reason}
+    end
+  end
+
+  defp sumValues(addends, payload) do
+    Enum.reduce_while(addends, {:ok, 0}, fn addend, {:ok, acc} ->
+      result =
+        cond do
+          is_number(addend) -> addend
+          true -> Map.get(payload, addend)
+        end
+
+      cond do
+        is_number(result) -> {:cont, {:ok, acc + result}}
+        is_nil(result) -> {:halt, {:error, "Missing field in payload: #{addend}"}}
+        true -> {:halt, {:error, "A value is not a number: #{addend}"}}
+      end
+    end)
+  end
+
+  def validate(parameters) do
+    %ValidationStatus{}
+    |> NotBlank.check(parameters, @addends)
+    |> NotBlank.check(parameters, @target_field)
+    |> ValidationStatus.ordered_values_or_errors([@addends, @target_field])
+  end
+
+  def fields() do
+    [
+      %{
+        field_name: "targetField",
+        field_type: "string",
+        field_label: "Field to populate with sum",
+        options: nil
+      },
+      %{
+        field_name: "targetField",
+        field_type: "list",
+        field_label: "List of values or fields to add to targetField",
```

Comment:
Since the labels end up on the UI, could I recommend something like "List of values or fields to add together"? Or include target field somehow in the label for that field. Just so it's clear to a user how those two relate. Hopefully it's obvious but DX isn't my strength.